IgOmeProfiling_pipeline.py

```diff
@@ -264,6 +265,7 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
     parser.add_argument('--prefix_length_in_clstr', default=20, type=int,
                         help='How long should be the prefix that is taken from the clstr file (cd-hit max prefix is 20)')
     parser.add_argument('--multi_exp_config_inference', type=str, help='Configuration file for inference motifs phase to run multi expirements')
+    parser.add_argument('--precent_random_hit_pssm', type=float, default=0.05, help='Precent of random hits per PSSM, for calculate cutoffs')

```

Comment:
The name and help isn't very indicative/explanatory.
What it does is taken the score to top percentile of random peptides and define it a cutoff score, i.e., peptide with score equal or greater than cutoff is considered as hit.

Parameter name should be something similar to `cutoff_random_peptitdes_percentile`.

This comment is applicable to all the code change in this PR